examples/head_or_tails.py

```diff
@@ -0,0 +1,16 @@
+import random
+
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="$")
+
+@bot.command()
+async def headortails(ctx, answer=None:
+    machine_choice = random.choice(["heads", "tails"])
+
+    if machine_choice == answer:
+        await ctx.send("Congratulations")
+    else:
+        await ctx.send("Sorry you lost")
```

Comment:
```suggestion
        await ctx.reply("Congratulations")
    else:
        await ctx.reply("Sorry you lost")
```

Reply:
Is that a new syntax? 🤔

Reply:
Added but still confused why it changed to reply.